src/language/translate.tsx

```diff
@@ -39,6 +39,8 @@ const normalizeValue = (value, key) => {
   }
 };
 
+const isNullOrUndefined = (value: any) => value == null;
```

Comment:
Can you add a comment here to say double equals checks for null and undefined so no one is confused in future

Reply:
or should I use this code instead to be more explicit :  `value === null || value === undefined` ?

Reply:
that is fine as well